tests/Microsoft.Bot.Builder.Tests/ShowTyping_MiddlewareTests.cs

```diff
@@ -19,7 +19,7 @@ public async Task ShowTyping_TestMiddleware_1_Second_Interval()
 
             await new TestFlow(adapter, async (context, cancellationToken) =>
             {
-                await Task.Delay(TimeSpan.FromMilliseconds(2800));
+                await Task.Delay(TimeSpan.FromMilliseconds(2300));
```

Comment:
Why from 2800 to 2300? Was this arbitrary? On the description says 

> time waited reduced by half

Reply:
I overlooked this value while writing the original PR, thanks for pointing it out.
I corrected the description and clarified that both values were chosen based on brute force testing.